src/Components/RegistrationView/Steps/FifthStep.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { PropTypes } from 'prop-types';
+import { faSyncAlt } from '@fortawesome/free-solid-svg-icons';
+import Button from '../../Button/Button';
+import submitRegistrationData from '../../../requests';
+
+function FifthStep({ formData, onSubmit }) {
+  const [backendConnectionError, setErrorState] = useState(false);
+  const [connectionMessage, setConnectionMessage] = useState('');
+  const messageClassName =
+    connectionMessage === 'Data submited. Redirecting to Login'
```

Comment:
Logic based on such a long sting is not a good idea. Especially that while translations will be introduced, this will need to be refactored. In most cases, we operate on numeric states

Reply:
It is changed. Please review it